From 42ddcfff2eb94055b35bc68d84ec5faafa982faf Mon Sep 17 00:00:00 2001
From: Christoph Reiter <reiter.christoph@gmail.com>
Date: Fri, 23 Apr 2021 19:24:33 +0200
Subject: [PATCH 31/N] CI: give the cygwin sync job explicit write permissions

So we can change the default repo permissions to read-only
---
 .github/workflows/sync-with-cygwin.yml | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/.github/workflows/sync-with-cygwin.yml b/.github/workflows/sync-with-cygwin.yml
index defcdf8..57bd30e 100644
--- a/.github/workflows/sync-with-cygwin.yml
+++ b/.github/workflows/sync-with-cygwin.yml
@@ -3,11 +3,14 @@ name: sync-with-cygwin
 # File: .github/workflows/repo-sync.yml
 
 on:
+  workflow_dispatch:
   schedule:
   - cron:  "42 * * * *"
 jobs:
   repo-sync:
     runs-on: ubuntu-latest
+    permissions:
+      contents: write
     steps:
     - name: Fetch Cygwin's latest master and tags
       run: |
-- 
2.31.1

